subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/ConsoleGradleBuildGroupedTaskFunctionalTest.groovy

```diff
@@ -45,12 +47,19 @@ class ConsoleGradleBuildGroupedTaskFunctionalTest extends AbstractConsoleGrouped
         file(externalBuildScriptPath) << externalBuildScript()
 
         when:
+        executer.withConsole(mode)
         succeeds(AGGREGATE_TASK_NAME)
 
         then:
         result.groupedOutput.task(':helloWorld').output == HELLO_WORLD_MESSAGE
         result.groupedOutput.task(":external:important").output == IMPORTANT_MESSAGE
         result.groupedOutput.task(':byeWorld').output == BYE_WORLD_MESSAGE
+        result.groupedOutput.hasTask(':silence') == hasSilenceTaskOutput
+
+        where:
+        mode    | hasSilenceTaskOutput
+        Rich    | false
+        Verbose | true
```

Comment:
Avoid highjacking test case to test other features. The Rich vs Verbose test doesn't belong to this test. Create a more focused test specifically for console mode with `GradleBuild` (if required by the test).

Reply:
Done.